libs/components/editor-blocks/src/components/source-view/format-url/affine.ts

```diff
@@ -1,5 +1,5 @@
 const _regex =
-    /^(https?:\/\/(localhost:4200|(nightly|app)\.affine\.pro|.*?\.ligo-virgo\.pages\.dev)\/\w{28}\/)?(affine\w{16})(\/whiteboard)?$/;
+    /^(https?:\/\/(localhost:4200|(nightly|app)\.affine\.pro|.*?\.ligo-virgo\.pages\.dev)\/\w{28}\/)?(affine\w{16})(\/edgeless)?$/;
```

Comment:
is the check too strict? @darkskygit

Reply:
domain can relax to current domain, fix it after the two-way link work correct